pkg/test/framework/components/mixer/attrmanifest.go

```diff
@@ -0,0 +1,52 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package mixer
+
+import (
+	"errors"
+	"strings"
+
+	"istio.io/istio/pkg/test/framework/repository"
+	"istio.io/istio/pkg/test/framework/scopes"
+	"istio.io/istio/pkg/test/helm"
+)
+
+// Extract attribute manifest from Helm charts.
+func extractAttributeManifestFromHelm() (string, error) {
```

Comment:
Is this mixer-specific? If not, maybe it should be in a more common place? Maybe something like `/pkg/test/kube/helm/...`?